processes/org-monthly-meetings/minutes/2019-06-Minutes.md

```diff
@@ -0,0 +1,173 @@
+
+The idea would be to start these workshops in September.
+
+**Owner:** @loriking
+
+**Status:**
+This initiative is **ongoing** and should be included in next month agenda.
+
+
+## Define process for selecting next hosts
+The host selection sheet has been completed. A section to add historical data needs to be included and all Barcelona team members need to complete the host rating. There is a possibility of adding more dates to the calendar, but this will not happen until September. There is a possibility of having a session in August, although it might be a more social session.
+
+**Owner:** @amaliacardenas
+
+**Status:**
+This initiative is **ongoing** and should be included in next month agenda.
+
+
+## Codebar landing page. Explanation in 3 languages
+David attended the monthly organizer's meeting and discussed why it is important for the Barcelona chapter to have an explanation in three languages. Some of our students do not speak English, and we need a way for them to feel welcome and safe during the workshops. There is a possibility to translate the entire website. The translation project can be turned into a workshop. It will be a good way to include our students and it's a great way to learn git. There is a UX designer from London who is working on redesigning the landing page so that the page can be in different languages. It will take some time for the redesign to be completed, but the project is progressing.  
+
+**Owner:** @jmbriano
+
+**Status:**
+This initiative is **ongoing** and should be included in next month agenda.
+
+
+## Adding collaborators board
+**Owner:** @jmbriano
+Martin was not available to give us an update.
+
+**Status:**
+This initiative is **ongoing** and should be included in next month agenda.
+
+
+## Pride Month
+The idea is to bring together different collectives from various companies and organize a social event. David, Raquel and Amalia will work together on this initiative. We will contact several people from Thoughtworks and Typeform to get the initiative going.
+
+**Owner:** @daverick
+
+**Status:**
+This initiative is **ongoing** and should be included in next month agenda.
+
+
+## On-boarding process document
+The on-boarding document has been created and has been merged. We also tested the document with Raquel and Salva during an on-boarding session. If the document needs to be updated, members can create a PR to update.
+
+**Owner:** @amaliacardenas
+
+**Status:**
+This initiative can be **closed**
+
+
+## E-mail templates
+Amalia will follow up with the London team to see if the Barcelona team can have codebar emails.
+
+As a group we identified that it can be confusing to know who i should answer an email if we get contacted by somebody. In order to make things more clear, we decided to add a "on call" field on the trello card for each workshop that we are organizing. The person who is on call for that week, is responsible for answering codebar emails.    
+
```

Comment:
This is a bit confusing, there are cards per event and one event is "under preparation" for two weeks.
I think mixing the "on call" responsible in the event cards is not correct. They are independent processes.

What we can do is to add a new column "on call", put one card per person, and put on top the person on call that week. After each week, the one on top goes to bottom and the next in line is "on call". Also we can do this in slack in our #barcelona-org channel "topic". 

Can we keep the implementation on hold and discuss it again next month?

Reply:
Another option is a simple reply to barcelona@codebar.io saying "I reply this tonight" or similar.

Reply:
Its a good point @jmbriano , while it does not change the functionality, the way to display it is more explicit, and it would help to prevent to forget defining the role for the workshop owner (like i did this week *cough* :sweat_smile: )

Reply:
about the reply to barcelona@codebar.io, my concern is that, not everyone pays attention 24/7 to the email (specially me :laughing: ) and if i don't see the email inmediatly, i can't do the reply, probably i'd see the email in the afternoon / night and just reply it directly

Reply:
the objectives is not to answer straight away but in a timely manner and that there is no collusion.
so sending another mail to barcelona@codebar.io won't work, the same as using slack. we are not online 24/7.
Having an new columns in the troll for on-call looks ok for me. Let's try it!